ocf/serviced

```diff
@@ -211,8 +211,10 @@ serviced_monitor() {
         check_serviced_exit_code
         exit_ret_val=$?
         if [ $exit_ret_val -ne 0 ]; then
+            ocf_log info "serviced_monitor - Control Center failed"
```

Comment:
So close. This should be `err` not info. Probably the same for the message at line 170 come to think of it